### PR TITLE
Modules - Buttons should be updated

### PR DESCRIPTION
**Pull request by khouloudbelguith**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Buttons should be updated
| Type?         | improvement
| BC breaks?    |no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/21361
| How to test?  | Follow https://github.com/PrestaShop/PrestaShop/issues/21361

### CODE REVIEW

File: ps_shoppingcart.php [hunk 1]
```diff
@@ -226,12 +226,12 @@ public function renderForm()
                             [
                                 'id' => 'active_on',
                                 'value' => 1,
-                                'label' => $this->trans('Enabled', [], 'Admin.Global'),
+                                'label' => $this->trans('YES', [], 'Admin.Global'),
```
Comment (PierreRambaud) [inline ps_shoppingcart.php:229]:
```suggestion
                                'label' => $this->trans('Yes', [], 'Admin.Global'),
```

File: ps_shoppingcart.php [hunk 2]
```diff
@@ -226,12 +226,12 @@ public function renderForm()
...
                                 'id' => 'active_on',
                                 'value' => 1,
-                                'label' => $this->trans('Enabled', [], 'Admin.Global'),
+                                'label' => $this->trans('Yes', [], 'Admin.Global'),
                             ],
                             [
                                 'id' => 'active_off',
                                 'value' => 0,
-                                'label' => $this->trans('Disabled', [], 'Admin.Global'),
+                                'label' => $this->trans('NO', [], 'Admin.Global'),
```
Comment (PierreRambaud) [inline ps_shoppingcart.php:234]:
```suggestion
                                'label' => $this->trans('No', [], 'Admin.Global'),
```

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by ngodefroy** [comment]
TEST OK
Capture d’écran 2020-10-09 à 13 27 44

**Comment by Progi1984** [comment]
Thanks @khouloudbelguith & @ngodefroy

**Comment by ngodefroy** [comment]
Hello @Progi1984 
So sorry, I just realized the confirmation message doesnt show up when you switch to YES / NO
https://drive.google.com/file/d/1PFwFV1R5z1_aIrsAc1m8BUxTa2f2cNdZ/view

**Comment by khouloudbelguith** [comment]
Hi @ngodefroy,

It is not related to this PR.
If you can check 1768, same issue.
https://drive.google.com/file/d/1ETB1yvOnpcF8yePolzorxb6_0gC1xlHF/view?usp=sharing

Thanks!

**Pull request merged**